src/components/App/index.spec.tsx

```diff
@@ -161,4 +162,51 @@ describe(__filename, () => {
 
     expect(dispatch).not.toHaveBeenCalled();
   });
+
+  it('does not render application errors when there is no error', () => {
+    const root = render();
+
+    expect(root.find(`.${styles.errors}`)).toHaveLength(0);
+  });
+
+  it('renders application errors', () => {
+    const error1 = new Error('error 1');
+    const error2 = new Error('error 2');
+    const store = configureStore();
+    store.dispatch(errorsActions.addError({ error: error1 }));
+    store.dispatch(errorsActions.addError({ error: error2 }));
+
+    const root = render({ store });
+
+    expect(root.find(`.${styles.errors}`)).toHaveLength(1);
```

Comment:
Since this style is not used visually, I don't think you need it. The call to `root.find(Alert)` is enough.